libs/blocks/aside/aside.css

```diff
@@ -550,10 +730,37 @@
     margin-top: 0;
   }
 
+  .aside.split.bio .foreground.container .text .icon-area {
+    display: block;
+  }
+
   .aside.notification.small .foreground.container .text .icon-area {
     width: 40px;
     height: 40px;
   }
+  
+  .aside.center:not(.notification):not(.split) .foreground.container .text .icon-area {
+    height: 80px;
+  }
+
+  .aside.center:not(.notification):not(.split) .foreground.container .text .icon-area img {
+    height: 80px;
```

Comment:
```suggestion
    height: val(--icon-size-xxl);
```

Reply:
@ryanmparrish I have updated the icon sizes as suggested to a standard icon-size variable name.